CODEOWNERS

```diff
@@ -12,5 +12,3 @@ build/python.props @vrdmr @gavin-aguiar @YunchuWang
 test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json @fabiocav @brettsam @mathewc
 
 src/WebJobs.Script.WebHost/PreJIT/* @vrdmr @pragnagopa @eliaslopezgt @VpOfEngineering @azure/azure-functions-core
-
-* @azure/azure-functions-core
```

Comment:
I am not completely aware of the intend of adding this line in the commit - [Link](https://github.com/Azure/azure-functions-host/pull/8812). Creating this PR to address Varad's [comment](https://github.com/Azure/azure-functions-host/pull/8812#issuecomment-1275232727) about the pattern being broken so that we can discuss here and fix the issue.

Reply:
I don't think we should remove it completely, you can move it up to the top (above line 8) so that the other filters are not invalid